### PR TITLE
Changes a bit how hardsuit jetpack upgrade works

### PR DESCRIPTION
**Pull request by Core0verload**
- Hardsuit jetpack upgrade item is replaced by an actual jetpack - the same as the one that exists inside the upgraded suit. It still can't be used separately.
- Hardsuit jetpack has hardsuit jetpack sprites instead of voidsuit pack.
- You can remove jetpack from hardsuit by using screwdriver on it.

Unrelated:
- Fixes mining vendor UI a bit: items with long names no longer take two lines.
- Changes staff of storms path to actual staff.

:cl: CoreOverload
add: You can remove jetpack from a hardsuit by using screwdriver on it.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/tanks/jetpack.dm [hunk 1]
```diff
@@ -110,28 +111,42 @@
...
 	STOP_PROCESSING(SSobj, src)
-	air_contents = null
+	temp_air_contents = air_contents
+
+/obj/item/weapon/tank/jetpack/suit/attack_self()
+	return
 
 /obj/item/weapon/tank/jetpack/suit/cycle(mob/user)
+	if(!istype(loc, /obj/item/clothing/suit/space/hardsuit))
+		user << "<span class='warning'>\The [src] must be connected to hardsuit!</span>"
```
Comment (RemieRichards) [inline code/game/objects/items/weapons/tanks/jetpack.dm]: ```to a hardsuit!```

**Comment by KorPhaeron** [comment]
Changelog

**Comment by Core0verload** [comment]
Was it really _that_ necessary?

**Comment by Core0verload** [comment]
conflicts are resolved, changelog is in, ready to merge

**Pull request merged**